### PR TITLE
[#714] calico cni

### LINKED ISSUE #627: Network layer of k8s is not stable need to update it
We face with network stability problems on our current network k8s version during multiple and fast operations with our test models like"scale, deploy, undeploy" legionctl commands.
Because of that, we have failed tests sometimes 
```
2018-11-26 02:19:14,100 - ERROR - Time out: operation has not been confirmed
Traceback (most recent call last):
  File "/var/lib/jenkins/workspace/Deploy_Legion_Release/legion/bin/legionctl", line 178, in <module>
    args.func(args)
  File "/var/lib/jenkins/workspace/Deploy_Legion_Release/legion/legion/edi/deploy.py", line 282, in scale_kubernetes
    lambda affected_deployments_status: check_all_scaled(affected_deployments_status,
  File "/var/lib/jenkins/workspace/Deploy_Legion_Release/legion/legion/edi/deploy.py", line 207, in wait_operation_finish
    raise Exception('Time out: operation has not been confirmed')
Exception: Time out: operation has not been confirmed
```

### PR DESCRIPTION
**Pull request by alexander-semenets**
KOPS and K8S version were updated for better compatibility with latest calico version
This closes #627 and #714

### CODE REVIEW

**Comment by alexander-semenets** [comment]
bot, test all

**Comment by alexander-semenets** [comment]
bot, test all

**Comment by kirillmakhonin** [comment]
bot, test all

File: deploy/ansible/roles/k8s_create_cluster/tasks/install-kops.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@
     group: root
     mode: 0755
     force: yes
-    sha256sum: "{{ kops_sha256sum }}"
+#    sha256sum: "{{ kops_sha256sum }}"
```
Comment (mcdoker18) [inline deploy/ansible/roles/k8s_create_cluster/tasks/install-kops.yaml]: Please, revert the checksum

File: deploy/ansible/roles/k8s_create_cluster/defaults/main.yaml [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+kops_sha256sum: "c4dfcb3f2b76f39b9b4fa3f784f4154077453d09"
 
-kubectl_version: "v1.10.6"
+kubectl_version: "v1.11.6"
 kubectl_download_path: "https://storage.googleapis.com/kubernetes-release/release/{{ kubectl_version }}/bin/linux/amd64"
 kubectl_darwin_download_path: "https://storage.googleapis.com/kubernetes-release/release/{{ kubectl_version }}/bin/darwin/amd64"
 
-aws_image: kope.io/k8s-1.9-debian-stretch-amd64-hvm-ebs-2018-05-27
+aws_image:  kope.io/k8s-1.11-debian-stretch-amd64-hvm-ebs-2018-08-17
+# kope.io/k8s-1.10-debian-jessie-amd64-hvm-ebs-2018-08-17
```
Comment (mcdoker18) [inline deploy/ansible/roles/k8s_create_cluster/defaults/main.yaml]: Remove commented lines

File: deploy/ansible/roles/k8s_create_cluster/templates/kops.yaml.j2 [hunk 3]
```diff
@@ -12,8 +12,8 @@ spec:
       additionalSecurityGroups: {{ external_access_sgs|to_yaml }}
   authorization:
     rbac: {}
-  kubeAPIServer:
-    authorizationRbacSuperUser: admin
+#  kubeAPIServer:
```
Comment (mcdoker18) [inline deploy/ansible/roles/k8s_create_cluster/templates/kops.yaml.j2]: Remove the commented lines

File: k8s/ansible/Dockerfile [hunk 4]
```diff
@@ -30,7 +30,7 @@ RUN apt-get install -y \
 RUN mkdir -p /etc/ansible/ && /bin/echo -e "[local]\nlocalhost ansible_connection=local" > /etc/ansible/hosts
 
 # Install Kops
-ENV KOPS_VERSION=1.10.0
+ENV KOPS_VERSION=1.11.0
```
Comment (mcdoker18) [inline k8s/ansible/Dockerfile:33]: Upgrade kubectl version too

**Review by mcdoker18 — CHANGES_REQUESTED** [review]

**Comment by alexander-semenets** [comment]
all comments have been addressed

**Comment by alexander-semenets** [comment]
bot, test all

**Review by mcdoker18 — APPROVED** [review]

**Review by kirillmakhonin — APPROVED** [review]

**Review by botsman — APPROVED** [review]

**Comment by alexander-semenets** [comment]
bot, test all

**Comment by alexander-semenets** [comment]
bot, test all

**Comment by alexander-semenets** [comment]
bot, test all

**Comment by alexander-semenets** [comment]
bot, test all

**Pull request merged**